site/docs/spark-procedures.md

```diff
@@ -183,6 +183,12 @@ and their files which are no longer needed.
 This procedure will remove old snapshots and data files which are uniquely required by those old snapshots. This means
 the `expire_snapshots` procedure will never remove files which are still required by a non-expired snapshot.
 
+Available [table behavior properties](./configuration#Table-behavior-properties) for `expire_snapshots`:
```

Comment:
I think that this should be shorter and not duplicate information at the linked location. Otherwise, we need to update it when the defaults change.

This should also be combined with the Usage section below because that contains defaults that are determined by these settings.

Instead of this paragraph, I would add one below the Usage table that states:

> If `older_than` and `retain_last` are omitted, the table's [expiration properties](./configuration#Table-behavior-properties) will be used.